vue/package-lock.json

```diff
@@ -0,0 +1,838 @@
+{
+  "name": "opossum-vue-example",
```

Comment:
this should be opossum-vue.   when i did the npm install locally, this got updated